src/test/java/reactor/kafka/AbstractKafkaTest.java

```diff
@@ -54,17 +54,22 @@
 import org.junit.rules.TestName;
 
 import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.TestcontainersConfiguration;
 import reactor.core.publisher.Flux;
 import reactor.kafka.receiver.ReceiverOptions;
 import reactor.kafka.sender.SenderOptions;
 import reactor.kafka.sender.SenderRecord;
 import reactor.kafka.util.TestUtils;
 
 public abstract class AbstractKafkaTest {
+    private static final String KAFKA_DOCKER_TAG = "5.5.1";
+    private static final DockerImageName KAFKA_DOCKER = TestcontainersConfiguration.
```

Comment:
let's inline this constant (not used anywhere except `KafkaContainer'`s constructor) and use `DockerImageName.parse("confluentinc/cp-kafka:5.5.1")` for the value